localnode/NODE_RUNNING.md

```diff
@@ -1,503 +1,7 @@
-* Issue: A PoP Miner connected to `BFG` is not getting any new keystones
-  * `op-node` is likely not peered correctly with the Hemi Consensus-Layer P2P Network, _and_ `prysm` is likely not
-    peered correctly with the Ethereum Consensus-Layer P2P Network
-* Issue: A PoP Miner connected to `BFG` is getting delayed keystones
-  * `op-node` is likely not peered correctly with the Hemi Consensus-Layer P2P Network (should also see `op -geth`
-    and `op-node` consistently behind the tip)
-* Issue: A PoP Miner connecting to `BFG` is getting keystones, but is not able to send Bitcoin transactions
-  * `bitcoind` is likely not peered correctly with the Bitcoin P2P Network (so `electrs` is unable to get updated UTXO
-    data)
-
-## Bring Your Own L1s
-
-> [!TIP]
-> This section is required if you are running the `hemi` or `hemi-min` (Docker Profiles)(#docker-profiles), as these
-> profiles do not contain self-hosted Bitcoin and Ethereum full nodes, and the Hemi stack requires these to function
-> properly.
-
-Several of the containers (the Bitcoin and Ethereum L1 containers) that are normally run as part of the `full` profile
-can be replaced with nodes or API endpoints of your own. These can either be 3rd party RPC endpoints, or RPC endpoints
-you are hosting yourself separately using the `L1` Docker Profile.
-
-To run just the L1 containers, start with this command:
-
-```sh
-docker compose -f localnode/docker-compose_mainnet.yml --profile L1 up --build
-```
-
-To run only the Hemi containers (either `hemi` or `hemi-min` Docker Profiles), add your endpoints to `.env`:
-
-```sh
-OPSYNCMODE=consensus-layer
-BITCOINENDPOINT=<BitcoinRPC>
-BITCOINCREDS=<BitcoinRPCCreds>
-GETHL1ENDPOINT=<EthereumExecutionRPCEndpoint>
-PRYSMENDPOINT=<EthereumBeaconRPCEndpoint>
-```
-
-and run (replace `hemi` with `hemi-min` if you want to run the minimal Hemi stack):
-
-```sh
-docker compose -f localnode/docker-compose_mainnet.yml --profile hemi up --build
-```
-
-and similarly for testnet (`docker-compose.yml` rather than `docker-compose_mainnet.yml`).
-
-> [!TIP]
-> If you are running the `hemi-min` stack, you do not need to provide a `BITCOINENDPOINT` or `BITCOINCREDS`, as
-> the `hemi-min` stack does not run `BFG` or `electrs` which require a `bitcoind` RPC connection.
-
-## Monitoring
-
-A simple script setup to monitor all the running daemons is provided as well for mainnet or testnet (just run the
-Docker compose command in the first pane). This requires tmux and is primarily for interactive use/debugging, not
-production running.
-
-```sh
-./heminode.sh
-```
-
-## Running Without Docker
-
-Steps to run the Hemi stack without Docker will be detailed at a later date.
+> **This documentation and the `localnode` directory have been moved to the
+dedicated [hemi-node](https://github.com/hemilabs/hemi-node) repository.**
+>
+> For the latest setup instructions and node files, please visit: https://github.com/hemilabs/hemi-node
```

Comment:
I think this file should point to the NODE_RUNNING file in the hemi-node repo directly.